npachpande/async/miawCustomPrechat/customPrechatFormField.js

```diff
@@ -0,0 +1,60 @@
+import { track, api, LightningElement } from "lwc";
+
+export default class CustomPrechatFormField extends LightningElement {
+    choiceListDefaultValue;
+
+    @api fieldInfo = {};
+
+    @api
+    get name() {
+        return this.fieldInfo.name;
+    }
+
+    @api
+    get value() {
+        const lightningCmp = this.isTypeChoiceList ? this.template.querySelector("lightning-combobox") : this.template.querySelector("lightning-input");
+        return this.isTypeCheckbox ? lightningCmp.checked : lightningCmp.value;
+    }
+
+    @api
+    reportValidity() {
```

Comment:
required for validating each formField on Start Conversation click. A customer can choose their own way of validating the form but I just thought we can show them an example.